### DISCUSSION TITLE
How to implement drag-and-drop between a custom tree view and the file explorer?

### DISCUSSION THREAD

**Discussion by donfour** [microsoft/vscode-discussions#581] [Extension Development QnA] [answered]
From this [SO post](https://stackoverflow.com/questions/72055681/is-it-possible-to-drag-drop-between-custom-vs-code-tree-views), it looks like it's possible to drag-and-drop between two custom tree views.

However, I'd like to implement a feature to support drag-and-drop between a custom tree view and the built-in file explorer (the place where you can see the currently open folder in VS Code).

I'm stuck because I'm not sure what the `mimeType` should be for the file explorer. Is it possible to implement such a feature?

Thanks!

**Comment by isidorn** [comment, marked as answer]
Currently this is not possible.
